index.js

```diff
@@ -59,6 +59,10 @@ function photon (imageUrl, opts) {
     params.pathname = parsedUrl.pathname;
     params.hostname = parsedUrl.hostname;
   } else {
+    // Photon does not support URLs with a querystring component
+    if (parsedUrl.search ) {
```

Comment:
Tiny: extra space here. I am assuming this package follows different coding standards.